.azure-pipelines.yml

```diff
@@ -152,10 +152,11 @@ jobs:
     parameters:
       packages: virtualenv
   - template: tools/ci/azure/install_certs.yml
+  - template: tools/ci/azure/install_edge.yml
   - template: tools/ci/azure/update_hosts.yml
   - template: tools/ci/azure/update_manifest.yml
-  - script: python ./wpt run --yes --no-manifest-update --install-fonts --manifest MANIFEST.json --metadata infrastructure/metadata/ --webdriver-arg=--verbose --log-tbpl $(Build.ArtifactStagingDirectory)/edge.tbpl.log --log-tbpl-level info edge_webdriver infrastructure/
```

Comment:
Is there any value to keep running Edge stable as well, as we do for the other browsers?

Reply:
We can keep it, but we haven't been able to get a good run from it, so i thought we should remove it. If we do keep it, how do you want us to distinguish the two runs?

Reply:
The value I thought it might have, to Microsoft, is checking what tests passed on EdgeHTML and will fail on Chromium-based Edge. You probably know the broad strokes of what things were better in EdgeHTML, but perhaps this would reveal some additional things worth triaging.

Although this configuration certainly hasn't been robust, sometimes it has produced pretty complete results, and just a few aligned runs like that would allow you to compare.

Whether keeping it is worthwhile is entirely up to you. If you do want to keep it for a while, we would distinguish them by label and version, just like for other browsers. We could configure stable to run daily or less frequently to not waste resources.

Reply:
After discussing this internally we've decided not to keep the EdgeHTML Edge runs going since we don't plan to make any changes it anymore and we'll have existing historic data if we need it for comparison.

Reply:
Alright, thanks for confirming! @jugglinmike FYI, this means that Edge from Sauce only matters until there are reliable Edgium runs from this setup, but comparing the results won't be meaningful.

Reply:
Got it. Thanks for the heads up!